### PR TITLE
EUI-3132-CAA-content-changes

### PR DESCRIPTION
**Pull request by ishOsw**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3132

### Change description ###
CAA content changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/users/components/invite-user-permissions/invite-user-permission.component.html [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
+          [isChecked]="isCaseAccessAdmin"
+          [config]="{value: 'pui-caa', label: 'Case access administrator', name: 'roles',
+          hint: 'Manage case access for all users within your organisation. You can assign this role to any user in your organisation. However, your service might not currently support this role. You will receive further guidance once it becomes available.'}">
       </lib-gov-checkbox>
+        <!-- <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiFinanceManager"
+          [config]="{value: 'pui-finance-manager', label: 'Manage Payments', name: 'roles',
+          hint: 'View account balances, available credit and transactions.'}">
+        </lib-gov-checkbox> -->
```
Comment (rayliangatos) [inline src/users/components/invite-user-permissions/invite-user-permission.component.html:40]: remove this if comment out

Comment (leepavlouhmcts) [reply src/users/components/invite-user-permissions/invite-user-permission.component.html:40]: agreed. if im not allowed to leave commented code, neither can you lol

**Review by leepavlouhmcts — APPROVED** [review]
approved

File: src/users/components/invite-user-permissions/invite-user-permission.component.html [hunk 2]
```diff
@@ -1,31 +1,47 @@
...
-        [config]="{value: 'pui-organisation-manager', label: 'Manage Organisation', name: 'roles',
-        hint: 'View organisation name and addresses.'}">
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiOrganisationManager"
+          [config]="{value: 'pui-organisation-manager', label: 'Manage organisation', name: 'roles',
+          hint: 'View the organisation name and addresses.'}">
+        </lib-gov-checkbox>
+
+        <lib-gov-checkbox
```
Comment (rayliangatos) [inline src/users/components/invite-user-permissions/invite-user-permission.component.html:29]: this seems like not synchronized from master, I can't see the latest change I made in another PR:
https://github.com/hmcts/rpx-xui-manage-organisations/pull/623
Could you double check what/how do you synchronize from master?

File: src/users/components/invite-user-permissions/invite-user-permission.component.html [hunk 3]
```diff
@@ -1,44 +1,38 @@
...
-        hint: 'View organisation name and addresses.'}">
-      </lib-gov-checkbox>
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiOrganisationManager"
+          [config]="{value: 'pui-organisation-manager', label: 'Manage organisation', name: 'roles',
+          hint: 'View the organisation name and addresses.'}">
+        </lib-gov-checkbox>
 
-      <lib-gov-checkbox id="checkbox-grant-case-access-admin" *ngIf="grantCaseAccessAdmin$ | async"
```
Comment (rayliangatos) [inline src/users/components/invite-user-permissions/invite-user-permission.component.html:29]: this logic needs to be preserved

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**